### DISCUSSION TITLE
Please set the correct MIME type for .wasm files

### DISCUSSION THREAD

**Discussion by dandv** [community/community#22863] [Other Feature Feedback, Questions, & Ideas]
<p>I’m trying to run a WebAssembly sample from MDN. It loads a raw <strong>.wasm</strong> file but runs into this error:</p>
<p><strong>Failed to execute ‘compile’ on ‘WebAssembly’: Incorrect response MIME type. Expected ‘application/wasm’.</strong></p>
<p>Here’s the fiddle: <a href="http://jsfiddle.net/dandv/oxt1476q/1/" rel="nofollow noopener">http://jsfiddle.net/dandv/oxt1476q/1/</a></p>
<p>Any chance for GitHub to set the MIME type correctly for .wasm files?</p>
<p>Thanks!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for reaching out about this! In short, it’s not possible to manually assign a content-type for files on GitHub Pages.</p>
<p>GitHub Pages supports more than 750 MIME types across 1,000s of file extensions. The list of supported MIME types is generated from the mime-db project, which aggregates MIME types from the Apache and Nginx projects as well as the official IANA list of internet content types.</p>
<p>If you wish to contribute a new mime type to mime-db please see the contributing instructions here.</p>
<p><a href="https://github.com/jshttp/mime-db#adding-custom-media-types" target="_blank" rel="nofollow noopener">https://github.com/jshttp/mime-db#adding-custom-media-types</a></p>
<p>I hope this helps!</p>

**Comment by Evidlo** [comment]
Is this fixed?  I checked jshttp and they seem to have `application/wasm` now, but I seem to get a `text/html` type on Github pages for my .wasm file.

https://evan.widloski.com/node_modules/sql.js/dist/sql-wasm.wasm
